### PR TITLE
[BULK UPDATE] DocuTune - Rebranding

### PR DESCRIPTION
**Pull request by alexbuckgit**
**Global effort for the Microsoft Learn platform rebrand**

The Microsoft team is applying changes to content for the Microsoft Learn platform rebrand. Changes include modifications to current Microsoft Learn and Microsoft Docs terminology as well as associated domain name and content URL path changes.

Please review this draft PR and provide any necessary comments as soon as possible. Because these fixes are tightly scoped, we have approval to automatically merge these changes to the default branch for publication.

This PR was generated via DocuTune. It includes only targeted fixes and minor formatting adjustments.

CorrelationId: d8c4c3ac-d49a-4c95-9ddc-9822ca297907
InitiativeId: docutune-rebranding-2022-09
PointOfContact: Alex Buck

### CODE REVIEW

**Comment by alexbuckgit** [comment]
#docutune-draft

File: docs/azure/deployment.md [hunk 1]
```diff
@@ -15,7 +15,7 @@ Whether your workflow is through the [Azure CLI](https://learn.microsoft.com/cli
 
 ## Deployment tutorials
 
-The tutorials below, hosted on the Microsoft Docs platform, walk you through different ways of creating and deploying apps to Azure using Visual Studio Code:
+These tutorials from Microsoft Learn show different ways of creating and deploying apps to Azure via Visual Studio Code:
```
Comment (gregvanl) [inline docs/azure/deployment.md:18]: Suggest not using "show" in order to be more inclusive to non-sighted readers.
If you think "walk you through" is too conversational, perhaps "teach you" or "describe".

Comment (alexbuckgit) [reply docs/azure/deployment.md:18]: Fixed

**Comment by alexbuckgit** [comment]
#docutune-ready

File: docs/azure/deployment.md [hunk 2]
```diff
@@ -15,7 +15,7 @@ Whether your workflow is through the [Azure CLI](https://learn.microsoft.com/cli
 
 ## Deployment tutorials
 
-The tutorials below, hosted on the Microsoft Docs platform, walk you through different ways of creating and deploying apps to Azure using Visual Studio Code:
+These tutorials from Microsoft Learn dscribe different ways of creating and deploying apps to Azure via Visual Studio Code:
```
Comment (gregvanl) [inline docs/azure/deployment.md:18]: typo

Comment (alexbuckgit) [reply docs/azure/deployment.md]: D'oh!  :-D

Fixed.

**Review by gregvanl — APPROVED** [review]

**Comment by alexbuckgit** [comment]
Hi @gregvanl - Please merge, thanks!

**Comment by gregvanl** [comment]
Thanks for working on this!

**Pull request merged**